etc/icinga2/conf.d/templates.conf

```diff
@@ -62,11 +68,16 @@ template Notification "mail-host-notification" {
 template Notification "mail-service-notification" {
   command = "mail-service-notification"
 
-  states = [ OK, Warning, Critical, Unknown ]
+  states = [ Critical, OK, Unknown, Warning ]
```

Comment:
That's the result of many many changes that ended up in "don't commit additional scripts, but replace the old ones". You can revert it, if you want (or I can).